### ISSUE TITLE
Any information on Testpads on the ZBD 50c?

### ISSUE THREAD

**Issue by xsrf** [atc1441/E-Paper_Pricetags#3]
Hey, do you have any information on the Testpads of the ZBD 50c? You did connect them to a programmer (?) in https://www.youtube.com/watch?v=OzS4UP7LHaE
I've mapped them but I haven't probed them...
Any useful signals while the display is wake/in sync? I assume you can use them for programming the CC1110?

<img src="https://user-images.githubusercontent.com/8081353/112719907-b2a49c80-8efb-11eb-93c2-68ee6a57f5e0.jpg" width="200" alt="zbd_50c_pcb_testpoints" title="zbd_50c_pcb_testpoints"/>

**Comment by xsrf** [comment]
From the datasheet: 
> The debug interface uses the I/O pins P2_1 as Debug Data and P2_2 as Debug Clock during Debug mode.
> The debug interface allows programming of the on-chip flash. It also provides access to memory and registers contents, and debug features such as breakpoints, single-stepping, and register modification.
> Debug mode is entered by forcing two rising edge transitions on pin P2_2 (Debug Clock) while the RESET_N input is held low.
So it's nice to see these broken out

**Comment by atc1441** [comment]
Yes these test pins are used to flash the CC1110, they also have the UART port brought out but until now it is not known what protocol it uses, i speculate on it being the RF data just directly via uart to set its function in production.

on all the ZBD and DisplayData displays these pins are available but not on all in such an order and over the whole pcb

Here is an not so nicely drawn overview
![20210327_131814](https://user-images.githubusercontent.com/1102694/112720600-a3bfe900-8eff-11eb-954f-974a48dca724.jpg)
![20210327_131818](https://user-images.githubusercontent.com/1102694/112720601-a4f11600-8eff-11eb-84a6-966ded4c9423.jpg)
![20210327_131840](https://user-images.githubusercontent.com/1102694/112720605-a6224300-8eff-11eb-9cc4-4a657861531c.jpg)
![20210327_131951](https://user-images.githubusercontent.com/1102694/112720606-a8849d00-8eff-11eb-9c93-09171f2db736.jpg)
